<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jsj_loginlongdlg</class>
 <widget class="QWidget" name="jsj_loginlongdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1481</width>
    <height>508</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1481</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(3, 59, 96);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1461</width>
     <height>401</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1461</width>
     <height>401</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1461</width>
     <height>1461</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(18, 81, 116, 180);
font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>356</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="UpBtn">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>49</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/上一页.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NextBtn">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>49</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>49</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/jsj/rc/用户管理/下一页.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>160</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>420</y>
     <width>685</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>31</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>月份选择</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>241</horstretch>
          <verstretch>41</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>241</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>241</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">

QComboBox{
color: rgb(255, 255, 255);
	font: 87 15pt &quot;Arial Black&quot;;
border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
}
QComboBox::down-arrow {

image:url(:/new/jsj/rc/新增/下拉箭头.png);

 }

 QComboBox::drop-down {

     border:none;

 padding-right:4px;

 }

QComboBox QAbstractItemView{

  outline:0px;

color:rgb(220,220,220);

background-color:rgb(37,86,159);

border:1px solid rgba(143,211,249,127);

border-bottom-right-radius:4px;

border-bottom-left-radius:4px;

 }

QComboBox QAbstractItemView::item{

height:26px;

background-color:rgba(143,211,249,38);

 }

 QComboBox QAbstractItemView::item:selected{

background-color:rgb(26,138,219);
 }</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>月登录时长:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>月登录次数:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>208</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 98, 0);
font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 98, 0);
font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
